### ISSUE TITLE
Tree View (facets) is not entering the correct mode with some common screen readers

### ISSUE THREAD

**Issue by zebdelk** [cedardevs/onestop#416]
A common combination is Firefox + NVDA, navigating with the down arrow to read each new element in the document.

When navigating from a filter heading (ie Data Theme) into the tree with the down arrow, the screen reader reports that it is on the treeview, but does not actually move the focus, preventing the user from interacting with the tree in any way. (tabbing into the tree works correctly).

See https://github.com/nvaccess/nvda/issues/3023
and
https://github.com/nvaccess/nvda/issues/3573
In short, it's a bug in NVDA that is unlikely to be fixed, but the suggested way to handle it (end of the 2nd link) is to set tab index to -1 on the tree itself, and giving it a focus event that sets the focus onto the correct child item.

Try this fix.

**Comment by ajakz** [comment]
👍
